src/main/java/com/mparticle/kits/AppseeKit.java

```diff
@@ -0,0 +1,219 @@
+package com.mparticle.kits;
+
+import android.annotation.SuppressLint;
+import android.annotation.TargetApi;
+import android.content.Context;
+import android.location.Location;
+import android.os.Build;
+import android.os.Handler;
+import android.os.Looper;
+import com.appsee.Appsee;
+import com.mparticle.MPEvent;
+import com.mparticle.MParticle;
+import com.mparticle.consent.ConsentState;
+import com.mparticle.identity.MParticleUser;
+import com.mparticle.internal.Logger;
+
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+/**
+ *
+ * This is an mParticle kit, used to extend the functionality of mParticle SDK. Most Kits are wrappers/adapters
+ * to a 3rd party SDK, primarily used to map analogous public mParticle APIs onto a 3rd-party API/platform.
+ *
+ *
+ * Follow the steps below to implement your kit:
+ *
+ *  - Edit ./build.gradle to add any necessary dependencies, such as your company's SDK
+ *  - Rename this file/class, using your company name as the prefix, ie "AcmeKit"
+ *  - View the javadocs to learn more about the KitIntegration class as well as the interfaces it defines.
+ *  - Choose the additional interfaces that you need and have this class implement them,
+ *    ie 'AcmeKit extends KitIntegration implements KitIntegration.PushListener'
+ *
+ *  In addition to this file, you also will need to edit:
+ *  - ./build.gradle (as explained above)
+ *  - ./README.md
+ *  - ./src/main/AndroidManifest.xml
+ *  - ./consumer-proguard.pro
+ */
+public class AppseeKit extends KitIntegration implements KitIntegration.EventListener, KitIntegration.IdentityListener, KitIntegration.UserAttributeListener {
```

Comment:
looks like you can remove `KitIntegration.UserAttributeListener` - you're not using it.